src/BSON/UTCDateTime.c

```diff
@@ -142,7 +142,11 @@ static bool php_phongo_utcdatetime_init_from_date(php_phongo_utcdatetime_t *inte
 
 	/* The following assignments use the same logic as date_format() in php_date.c */
 	sec = datetime_obj->time->sse;
+#if TIMELIB_VERSION >= 201705
+	usec = (int64_t) floor(datetime_obj->time->us);
```

Comment:
It should be either a `int64_t` or `signed long long` per https://github.com/php/php-src/blob/21cd57e5f2d286ab77ab44a05d2d16c16e4f15e6/ext/date/lib/timelib.h#L124